test/specs/commonTests/isConformant.tsx

```diff
@@ -58,6 +61,14 @@ export default (Component, options: Conformant = {}) => {
         component = component.childAt(0) // skip the additional wrap <div> of the FocusZone
       }
     }
+
+    if (usesWrapperSlot) {
```

Comment:
@kuzhelov @levithomason 
I think we should avoid changing `isConformant` tests for the elements with wrapper slot; we had this discussion for `Input` component.

@miroslavstastny AFAIR, the tests that fail are related to `as` and `className` props not being set correctly; these can be solved by addressing the comments I had in `MenuItem.tsx` where `as` and `styles` should always be passed to the root element, being that the `a` or the `li` for the `MenuItem` case.

This probably requires a session where we can discuss options in more detail; @miroslavstastny feel free to schedule a meeting with me and @kuzhelov

Reply:
Thank you very much for the review, @Bugaa92.

So the main question is whether  we should split props between `li` and `a` as we do in Input or pass everything to the `a`.

I am not strictly against doing the split, but when I was considering this I thought it would make more sense to do 'everything to `a`' although it is different approach than in `Input`.

We should consider usability and API consistency first and test after that.

Agree to discuss this offline.